### PR TITLE
Homework 4 - Added validation checks using if statements to code from homework 3

### PR DESCRIPTION
**Pull request by BrunoMartinsons**
Here is my completed homework 4, it passes all the tests. Please let me know if anything can be improved.

Thank you in advance.

### CODE REVIEW

File: Src/BootCamp.Chapter/Checks.cs [hunk 1]
```diff
@@ -17,25 +17,25 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs:19]: |Comments can be removed

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -9,6 +9,122 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Need to refactor the code I made in Lesson 2.
+            // Use functions to achieve DRY (as little duplicate code as possible).
+
+            // Name input
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:15]: Comments can be removed, the functions speak for themselves.

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -9,6 +9,122 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Need to refactor the code I made in Lesson 2.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:12]: Second person is missing

File: Src/BootCamp.Chapter/Checks.cs (hunk 1 above)
Comment (BrunoMartinsons) [reply Src/BootCamp.Chapter/Checks.cs:19]: Removed comments and added second person.

File: Src/BootCamp.Chapter/Program.cs (hunk 2 above)
Comment (BrunoMartinsons) [reply Src/BootCamp.Chapter/Program.cs:15]: Comments have been removed.

File: Src/BootCamp.Chapter/Program.cs (hunk 3 above)
Comment (BrunoMartinsons) [reply Src/BootCamp.Chapter/Program.cs:12]: Added second person.

**Pull request closed without merging**